src/components/molecules/AdminSpaceCard/AdminSpaceCard.tsx

```diff
@@ -20,29 +30,85 @@ export interface AdminSpaceCardProps {
 }
 
 export const AdminSpaceCard: React.FC<AdminSpaceCardProps> = ({ venue }) => {
+  const [validBannerImageUrl] = useValidImage(
+    venue?.config?.landingPageConfig.bannerImageUrl,
+    DEFAULT_VENUE_BANNER_COLOR
+  );
+
   const backgroundStyle = useCss({
-    "background-image": `url(${venue.mapBackgroundImageUrl})`,
+    background: `linear-gradient(
+        rgba(25, 24, 26, 0.8),
+        rgba(25, 24, 26, 0.8)
+      ), url(${validBannerImageUrl})`,
   });
   const backgroundClasses = classNames("AdminSpaceCard__bg", backgroundStyle);
-
   const logoStyle = useCss({
-    "background-image": `url(${venue.host?.icon ?? DEFAULT_VENUE_LOGO})`,
+    "background-image": `url(${venue.host?.icon || DEFAULT_VENUE_LOGO})`,
   });
   const logoClasses = classNames("AdminSpaceCard__logo", logoStyle);
 
+  const spaceIcon = SPACE_PORTALS_ICONS_MAPPING[venue.template];
+
+  const spaceDescriptionText =
+    venue.config?.landingPageConfig?.description ||
+    "Description can be changed in space settings";
+
   return (
     <div className="AdminSpaceCard">
-      <div className={backgroundClasses} />
-      <div className="AdminSpaceCard__info">
-        <div className={logoClasses} />
-        <AdminShowcaseSubTitle>{venue.name}</AdminShowcaseSubTitle>
+      <div className={backgroundClasses}>
+        <div className="AdminSpaceCard__bg-container">
+          <Link
+            className="AdminSpaceCard__link"
+            to={venueInsideUrl(venue.slug)}
+            target="_blank"
+            rel="noopener noreferer"
+          >
+            {"Visit"}
+            <FontAwesomeIcon
+              className="AdminSpaceCard__link-icon"
+              icon={faExternalLinkAlt}
+            />
+          </Link>
```

Comment:
[NOTE]

`ButtonNG` has also the ability to create external links:

```tsx
      <ButtonNG
        isLink
        newTab
        linkTo={venueInsideUrl(venue.slug)}
      >
```

Reply:
that `ButtonNG` even with a `isLink` attribute differs greatly from the design, so I didn't use it